301-multi-tier-loadbalancing/azuredeploy.json

```diff
@@ -0,0 +1,1073 @@
+﻿{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "Region1-location-name": {
+      "defaultValue": "Central US",
+      "allowedValues": [
```

Comment:
It's probably best to not have allowedValues for regions for 2 reasons:

1) different subscriptions will have different lists of regions they can use; listing all of them invites the user to accidentally choose a region they can't deploy to.
2) the list changes over time :)